ambuda/utils/cheda.py

```diff
@@ -18,56 +18,56 @@ class Token:
 
 
 POS = {
-    "n": "noun",
-    "a": "adjective",
-    "v": "verb",
-    "i": "indeclinable",
-    "va": "participle",
-    "vi": "verbal indeclinable",
+    "n": {"en": "noun", "sa": "subanta"},
```

Comment:
Sanskrit conventions here are to use the nominative singular -- *subantam*, *vibhaktiḥ*, etc.